### PR TITLE
Add Pulumi AI and Registry blog post

### PR DESCRIPTION
**Pull request by AaronFriel**
## Description

Adds a blog post on enhancing Pulumi AI with Pulumi Registry documentation.

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.

### CODE REVIEW

**Review by scottslowe — APPROVED** [review]
LGTM!

File: themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+>     },
+>   },
+> });
+>
+> export const alloyDBClusterName = alloydbCluster.name;
+> ```
+
+## Getting Started with the Enhanced Pulumi AI
+
+Our enhancements to Pulumi AI have made it a more useful tool for generating accurate and relevant cloud infrastructure solutions. We're excited about the future of Pulumi AI as we continue to integrate it into [Pulumi Cloud](https://app.pulumi.com) and empower everyone to become a cloud engineer.
```
Comment (lukehoban) [inline themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md:147]:
```suggestion
Our enhancements to Pulumi AI have made it an even more useful tool for generating accurate and relevant cloud infrastructure solutions. We're excited about the future of Pulumi AI as we continue to integrate it into [Pulumi Cloud](https://app.pulumi.com) and empower everyone to become a cloud engineer.
```

File: themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+These enhancements are live now at [Pulumi AI](/ai/), and we've added links to Pulumi AI in pages on the Pulumi Registry, for example on the [`eks.Cluster`](/registry/packages/eks/api-docs/cluster/) docs mentioned earlier.
```
Comment (lukehoban) [inline themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md:149]:
```suggestion
These enhancements are live now at [Pulumi AI](/ai/).  We've also added "Explore with Pulumi AI" links to Pulumi Registry pages, for example on the [`eks.Cluster`](/registry/packages/eks/api-docs/cluster/) page mentioned earlier.
```

File: themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+Give them a try and share your experiences with us. Together, let's build a future where infrastructure as code becomes more accessible, efficient, and enjoyable for everyone.
```
Comment (lukehoban) [inline themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md:151]: Should we mention that we also added the ability to share your conversation results with others?  Perhaps include a screenshot like this:

Screenshot 2023-05-17 at 9 53 28 PM

**Review by lukehoban — APPROVED** [review]

File: themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md (hunk 3 above)
Comment (AaronFriel) [reply themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md]: Added

**Pull request merged**